Utils/azuretools-core/src/com/microsoft/azuretools/azurecommons/util/FileUtil.java

```diff
@@ -105,4 +107,37 @@ public static boolean isNullOrEmpty(final String name) {
 		return isValid;
 	}
 
+    /**
+     * Utility method to zip a file.
+     * @param inputStream
+     * @param fileName
+     * @return zip file
+     * @throws IOException
+     */
+    public static File zipFile(InputStream inputStream, String fileName) throws IOException {
```

Comment:
Can we use third-party lib to handle this? For example: https://github.com/zeroturnaround/zt-zip

Reply:
Adding external dependency is not suggested and it needs a complicated process. All zip related operations are provided by util functions in this project.

Reply:
> ZipUtil.pack(new File("/tmp/demo"), new File("/tmp/demo.zip"));

I think the signature `(File src, File dst)` is more clear if I want to compress some files. Is it possible to do in that way?